src/main/java/Duke.java

```diff
@@ -1,10 +1,111 @@
+import java.util.Scanner;
+
 public class Duke {
-    public static void main(String[] args) {
-        String logo = " ____        _        \n"
-                + "|  _ \\ _   _| | _____ \n"
-                + "| | | | | | | |/ / _ \\\n"
-                + "| |_| | |_| |   <  __/\n"
-                + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
-    }
+	public static void main (String[] args) {
+		printStart ();
+
+		Scanner scan = new Scanner (System.in);
+		String input = scan.nextLine ();
+		String exit = "bye";
+		String list = "list";
+		Task tasks[] = new Task[100];
+		int count = 0;
+
+		while (!(input.equalsIgnoreCase (exit)) && !(input.isEmpty ())) {
+
+			if (input.equalsIgnoreCase (list)) {
+				listTasks (tasks, count);
+			} else if (input.startsWith ("mark") || input.startsWith ("unmark")) {
+				changeStatus (input, tasks);
+			} else if (input.startsWith ("todo") || input.startsWith ("deadline") || input.startsWith ("event")) {
+				count = addTask (input, tasks, count);
+			} else {
+				break;
+			}
```

Comment:
Good K&R style brackets for your if-else statements!

Reply:
Thank you for your review! @ong-ck